### DISCUSSION TITLE
登录阿里云显示涉嫌违规，这是什么情况

### DISCUSSION THREAD

**Discussion by brianler** [messense/aliyundrive-webdav#739] [Q&A] [answered]
无法登陆阿里云，显示以下信息：
尊敬的用户你好，该账户因涉嫌违规，暂时无法使用阿里云盘服务，如有问题请联系客服邮箱：adrive@service.alibaba.com

**Comment by messense** [comment, marked as answer]
> 如有问题请联系客服邮箱：[adrive@service.alibaba.com](mailto:adrive@service.alibaba.com)
